src/main/java/pingis/utils/JavaClassGenerator.java

```diff
@@ -31,7 +51,7 @@ private static String indentateTaskAsMethod(List<Task> tasks) {
         String codeSegment = "";
         
         for (Task task : tasks) {
-            String[] lines = task.getCode().split(System.getProperty("line.separator"));
+            String[] lines = task.getCodeStub().split(System.getProperty("line.separator"));
```

Comment:
`codeStub` seems to always be an array in JSON, but contains either everything on a single line or each line as a separate string. Why?